examples/polls-demo/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "apollo-next-13-demo",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@apollo/experimental-nextjs-app-support": "workspace:^",
```

Comment:
@patrick91 can we have here the actual dependency, not the internal monorepo package?